packages/app/src/cli/commands/app/execute.ts

```diff
@@ -0,0 +1,27 @@
+import {appFlags} from '../../flags.js'
+import AppUnlinkedCommand, {AppUnlinkedCommandOutput} from '../../utilities/app-unlinked-command.js'
+import {AppInterface} from '../../models/app/app.js'
+import {globalFlags} from '@shopify/cli-kit/node/cli'
+import {renderSuccess} from '@shopify/cli-kit/node/ui'
+
+export default class Execute extends AppUnlinkedCommand {
```

Comment:
I think @nickwesselman mentioned something about marking the command as "hidden" -- should we do that to help prevent confusion for users while this is being built?